addons/languages/coptic/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
Since this is a new pack, you should set this to `false`. There is a bit of a process to get packs published.

Reply:
Thanks for taking a look. I've flipped the publication flag.